packages/pyo3/pyproject.toml

```diff
@@ -2,25 +2,29 @@
 name = "graspologic-native"
 maintainer = "Dax Pryce"
 maintainer-email = "daxpryce@microsoft.com"
-requires-python = ">=3.6,<3.13"
+requires-python = ">=3.8,<3.14"
 classifier = [
     "Development Status :: 5 - Production/Stable",
     "License :: OSI Approved :: MIT License",
     "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Python :: 3.6",
-    "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     "Topic :: Scientific/Engineering :: Mathematics"]
 
 [project.urls]
 Github = "https://github.com/microsoft/graspologic-native"
 Graspologic = "https://github.com/microsoft/graspologic"
 
-
 [build-system]
 requires = ["maturin>=1.4,<2.0"]
 build-backend = "maturin"
+
+[tool.uv]
+dev-dependencies = [
+    "ipython>=8.12.3",
+    "networkx==3",
```

Comment:
Will this pin to 3.0.0 or will it allow minor/patch bumps?

Reply:
I'm not sure, so I'll just be explicit and use `>=3,<4`